src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureTests.Fields.cs

```diff
@@ -21,8 +21,8 @@ namespace Preview_Feature_Scratch
 
             class Program
             {
-                private Dictionary<int, PreviewType> {|#0:_genericPreviewFieldDictionary|};
-                private List<List<List<List<PreviewType>>>> {|#1:_genericPreviewField|};
+                private Dictionary<int, {|#0:PreviewType|}> _genericPreviewFieldDictionary;
+                private List<List<List<List<{|#1:PreviewType|}>>>> _genericPreviewField;
                 private List<List<AGenericClass<Int32>>> _genericClassField;
                 private List<List<AGenericPreviewClass<Int32>>> {|#2:_genericPreviewClassField|};
```

Comment:
Generic types containing nested preview types are not handled yet. Filed https://github.com/dotnet/roslyn-analyzers/issues/5428 to address this.

Reply:
This is fixed now. Will add a comment to the PR later with the changes I made to fix this so Buyaa can see the latest commit